src/passes/StackCheck.cpp

```diff
@@ -135,13 +117,28 @@ struct EnforceStackLimits : public WalkerPass<PostWalker<EnforceStackLimits>> {
 };
 
 struct StackCheck : public Pass {
+  // The base is where the stack begins. As it goes down, that is the highest
+  // valid address.
+  Name STACK_BASE;
+  // The limit is the farthest it can grow to, which is the lowest valid
+  // address.
+  Name STACK_LIMIT;
+  // Exported function to set the base and the limit.
+  Name SET_STACK_LIMITS;
```

Comment:
These probably shouldn't be in ALL_CAPS anymore if they are no longer fixed.   

I guess `SET_STACK_LIMITS` can stay as a global since we are going to error out on conflict?